examples/opensora_hpcai/opensora/pipelines/train_pipeline.py

```diff
@@ -182,12 +201,16 @@ def compute_loss(
         fps: Optional[Tensor] = None,
         ar: Optional[Tensor] = None,
     ):
-        t = ops.randint(0, self.diffusion.num_timesteps, (x.shape[0],))
-        noise = ops.randn_like(x)
+        if self.mode == ms.PYNATIVE_MODE:
```

Comment:
why change this?

Reply:
Refer to https://github.com/mindspore-lab/mindone/pull/585#discussion_r1671445661